### DISCUSSION TITLE
2-Channel Wi-Fi AC Relay Module In Home Assistant | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1982] [Q&A]
# 2-Channel Wi-Fi AC Relay Module In Home Assistant | Seeed Studio Wiki

A smart device from Seeed Studio designed to control AC-powered appliances wirelessly

https://wiki.seeedstudio.com/2_channel_wifi_ac_relay/

**Comment by phillipthelen** [comment]
Is the esphome configuration installed by default available for this?

Alternatively more information on the pin configuration and what chip is used for the current sensing would also be useful to allow for custom configs

**Reply by nfs0619** [reply]
Thanks for reaching out! We recommend contacting our technical support team at [techsupport@seeed.io](mailto:techsupport@seeed.io). They will be able to provide you with more detailed guidance on your setup.

**Comment by discodyer** [comment]
The GitHub repository for this module has gone 404 and is inaccessible. 😭

**Reply by nfs0619** [reply]
Hello, thank you for your feedback. We have fixed the incorrect link in the documentation. The correct link is:
https://github.com/Seeed-Studio/xiao-esphome-projects/tree/main/projects/xiao_2_channel_wifi_ac_energy_meter
